pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -172,7 +172,7 @@ func (r *ReconcileApplicationLayer) Reconcile(ctx context.Context, request recon
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling ApplicationLayer")
 
-	applicationLayer, err := getApplicationLayer(ctx, r.client)
```

Comment:
I renamed this variable as `alp` because I wasn't having such a great time scanning the code because there's a package name referenced '`applicationlayer`' and there's this variable that was named '`applicationLayer`' with basically just a capitalisation the difference 😭

Reply:
Another option would be to switch its name to be `instance`. At least about half of the controllers fetch their main controlling CR as `instance`. If you don't want to switch to instance (which I'm not totally sold on as a good name) I'd at least prefer to switch to `al` as the `p` for policy is just one aspect that the resource controls.

Reply:
@tmjd great suggestion! i went with that, and will keep that in mind moving forward